pkgs/development/python-modules/cmsis-pack-manager/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, fetchPypi
+, rustPlatform
+, writeShellScriptBin
+, buildPythonPackage
+, setuptools-scm
+, appdirs
+, milksnake
+, pyyaml
+, hypothesis
+, jinja2
+, mock
+, pytestCheckHook
+}:
+let
+  pname = "cmsis-pack-manager";
+  version = "0.4.0";
+  name = pname + "-" + version;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-NeUG6PFI2eTwq5SNtAB6ZMA1M3z1JmMND29V9/O5sgw=";
+  };
+
+  native = rustPlatform.buildRustPackage {
+    name = name + "-native";
+
+    inherit src;
+
+    sourceRoot = name + "/rust";
+    cargoLock.lockFile = ./Cargo.lock;
+
```

Comment:
This needs `darwin.apple_sdk.frameworks.Security` added to `buildInputs` here on darwin.